test/parallel/test-worker-unref-from-message-during-exit.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
+const { Worker } = require('worker_threads');
+
```

Comment:
A note one what to expect if this test fails would likely be helpful as it's a bit subtle.

Reply:
@jasnell I’ve added a comment :+1: